packages/js/README.md

```diff
@@ -41,14 +41,16 @@ Before you're able to use this Matomo Tracker you need to initialize Matomo with
 const MatomoInstance = new window.MatomoTracker({
   urlBase: 'https://LINK.TO.DOMAIN',
   siteId: 3, // optional, default value: `1`
-  userId: 'UID76903202', // optional, default value: `undefined`.
   trackerUrl: 'https://LINK.TO.DOMAIN/tracking.php', // optional, default value: `${urlBase}matomo.php`
   srcUrl: 'https://LINK.TO.DOMAIN/tracking.js', // optional, default value: `${urlBase}matomo.js`
   heartBeat: { // optional, enabled by default
     active: true, // optional, default value: true
     seconds: 10 // optional, default value: `15
   }
-  linkTracking: false // optional, default value: true
+  linkTracking: false, // optional, default value: true
+  configurations: { // optional, default value: {}
+    // any valid matomo configuration
```

Comment:
Could you add an example of a configuration here?

Reply:
For exmaple, like so:

```
const matomoInstance = createInstance({
        urlBase: 'https://matomo.example.com',
        siteId: example_id,
        configurations: [
            ['setUserId', 'example_id'],
            ['disableCookies', true]
        ]
    })
```

Alternatively, it could be an Array of objects. The official syntax for disabling cookies is:

`window._paq.push(['disableCookies']);`

So maybe, it would be nice to make the second element of the array optional.

Andreas

Reply:
@fcandi With this code the current way is like this:
```js
const matomoInstance = createInstance({
    urlBase: 'https://matomo.example.com',
    siteId: example_id,
    configurations: {
        setUserId: 'example_id',
        disableCookies: true,
        setSecureCookie: true,
        setRequestMethod: 'POST',
    }
});
```

Reply:
Ok, its nice this way 👍

Reply:
@An-dz Nice! could you add that to the readme?